### PR TITLE
[stable/sonarqube] Support plugins that need copying to lib

### PR DESCRIPTION
**Pull request by stevehipwell**
#### Is this a new chart
No

#### What this PR does / why we need it:
Adds support for the [sonarqube-community-branch-plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin) which needs copying to `lib/common`.

#### Which issue this PR fixes
N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/sonarqube/values.yaml [hunk 1]
```diff
@@ -138,6 +138,7 @@ persistence:
 #    - "https://github.com/SonarSource/sonar-ldap/releases/download/2.2-RC3/sonar-ldap-plugin-2.2.0.601.jar"
 plugins:
   install: []
+  list: []
```
Comment (rjkernick) [inline stable/sonarqube/values.yaml]: This doesn't match the file above. Please change to 
`plugins:
    lib: []
`

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: stable/sonarqube/values.yaml (hunk 1 above)
Comment (stevehipwell) [reply stable/sonarqube/values.yaml]: Thanks, @rjkernick!

**Comment by stevehipwell** [comment]
@rjkernick could you mark this as ready to test?

**Comment by stevehipwell** [comment]
@zanhsieh @rjkernick could someone add the _lgtm_ label so we can get this merged?

**Comment by stevehipwell** [comment]
@rjkernick I've resolved the merge conflict. Could you add a `/lgtm`?

**Comment by stevehipwell** [comment]
@rjkernick I've resolved the new v7.9.2 merge conflict. Could you please add a `/lgtm` so we can get this out of the way and support the community branch plugin?

**Pull request merged**